src/main/java/org/gridsuite/modification/server/entities/equipment/modification/LinesAttachToSplitLinesEntity.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.equipment.modification;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.LinesAttachToSplitLinesInfos;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * @author bendaamerahm <ahmed.bendaamer at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "LinesAttachToSplitLines")
+public class LinesAttachToSplitLinesEntity extends ModificationEntity {
+
+    @Column
+    private String lineToAttachTo1Id;
+
+    @Column
+    private String lineToAttachTo2Id;
+
+    @Column
+    private String attachedLineId;
+
+    @Column
+    private String voltageLevelId;
+
+    @Column
+    private String bbsBusId;
```

Comment:
You could have kept bbsOrBusId, has in LineAttachToVoltageLevelEntity and LineSplitWithVoltageLevelEntity.

Reply:
i change it with many others to avoid code duplication issues